spec/unit/plugins/solaris2/network_spec.rb

```diff
@@ -143,7 +146,7 @@
     end
 
     it "detects the interfaces" do
-      expect(@plugin["network"]["interfaces"].keys.sort).to eq(["e1000g0:3", "e1000g2:1", "eri0", "ip.tun0", "ip.tun0:1", "ip6.tun0", "lo0", "lo0:3", "net0", "net1:1", "qfe1", "vni0"])
+      expect(@plugin["network"]["interfaces"].keys.sort).to eq(["e1000g0:3", "e1000g2:1", "eri0", "ip.tun0", "ip.tun0:1", "ip6.tun0", "ipmp0", "lo0", "lo0:3", "net0", "net1:1", "qfe1", "vni0"])
```

Comment:
Out of band, but this is a place where `%w[...]` is really nice:

```ruby
      expect(@plugin["network"]["interfaces"].keys.sort).to eq(%w[e1000g0:3 e1000g2:1 eri0 ip.tun0 ip.tun0:1 ip6.tun0 ipmp0 lo0 lo0:3 net0 net1:1 qfe1 vni0])
```

Reply:
that should be enforced and autofixed by chefstyle, i'm not sure why its not catching it here.